backend/dev/app.ts

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = require('./router.js');
```

Comment:
"imports" syntax  - more native for typescript

Reply:
Changed to:
`import express from "express";
import { router } from './router.js';`